packages/expo-updates/e2e/__tests__/fixtures/project_files/e2e/tests/utils/update.js

```diff
@@ -4,54 +4,90 @@ const fs = require('fs/promises');
 const path = require('path');
 
 const STATIC_FOLDER_PATH = path.resolve(__dirname, '..', '.static');
-const EXPORT_PUBLIC_URL = 'https://u.expo.dev/dummy-url';
+const RUNTIME_VERSION = '1.0.0';
+const server_host = process.env.UPDATES_HOST;
+const server_port = parseInt(process.env.UPDATES_PORT || '', 10);
```

Comment:
```suggestion
const serverHost = process.env.UPDATES_HOST;
const serverPort = parseInt(process.env.UPDATES_PORT || '', 10);
```

Reply:
These fixtures run in the test Detox app, which is not a Typescript app. I'm fine to convert it to Typescript, but that is worth a separate PR, I think.

Reply:
Variable names have been changed as requested